tests/testsuite/cargo_remove/gc_profile/in/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "cargo-remove-test-fixture"
+version = "0.1.0"
+
+[[bin]]
+name = "main"
+path = "src/main.rs"
+
+[build-dependencies]
+semver = "0.1.0"
+
+[dependencies]
+docopt = "0.6"
+rustc-serialize = "0.4"
+semver = "0.1"
+toml = "0.1"
+clippy = "0.4"
+
+[dev-dependencies]
+regex = "0.1.1"
+serde = "1.0.90"
+docopt = "0.6"
+
+[features]
+std = ["serde/std", "semver/std"]
+
+[profile.dev.package.docopt]
+opt-level = 3
+
+[profile.dev.package.toml]
+opt-level = 3
+
+[profile.release.package.toml]
+opt-level = 1
+overflow-checks = false
```

Comment:
Should we have a test that specifies a version?

Reply:
We should! I've done so for the `gc_profile` and `gc_replace` tests. Similarly, I've modified the test for `gc_patch` to include a second patch section which should not be removed.